x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -12,6 +12,7 @@ import {
   EuiTitle
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
+import { Location } from 'history';
```

Comment:
I'm using `Location` from `history` everywhere. Note: this is similar but not exactly the same as `Location` from the browser.